sections/dashboard/Stake/TradingRewardsTab.tsx

```diff
@@ -21,150 +23,152 @@ import {
+		);
+		const estimatedReward = useMemo(
+			() => BigNumber.from(estimatedRewardQuery?.data?.claims[walletAddress!]?.amount ?? 0),
+			[estimatedRewardQuery?.data?.claims, walletAddress]
+		);
+		const weeklyRewards = useMemo(
+			() => BigNumber.from(estimatedRewardQuery?.data?.tokenTotal ?? 0),
+			[estimatedRewardQuery?.data?.tokenTotal]
+		);
+
+		const claimDisabled = useMemo(() => totalRewards.lte(0), [totalRewards]);
+
+		const handleClaim = useCallback(() => {
+			dispatch(claimMultipleRewards());
+		}, [dispatch]);
+
+		const ratio = useMemo(() => {
+			return wei(weeklyRewards).gt(0) ? wei(estimatedReward).div(wei(weeklyRewards)) : zeroBN;
+		}, [estimatedReward, weeklyRewards]);
+
+		const showEstimatedValue = useMemo(() => wei(period).eq(epochPeriod), [epochPeriod, period]);
+
+		return (
+			<TradingRewardsContainer>
+				<CardGridContainer>
+					<CardGrid>
 						<div>
-							<WithCursor cursor="help">
-								<div className="title">
-									{t('dashboard.stake.tabs.trading-rewards.future-fee-paid', {
-										EpochPeriod: period,
-									})}
-								</div>
-								<div className="value">
-									{formatDollars(futuresFeePaid, { minDecimals: 2 })}
-									<HelpIcon />
-								</div>
-							</WithCursor>
+							<div className="title">
+								{t('dashboard.stake.tabs.trading-rewards.claimable-rewards-all')}
+							</div>
+							<LogoText yellow>{truncateNumbers(totalRewards, 4)}</LogoText>
 						</div>
-					</CustomStyledTooltip>
-					<div>
-						<div className="title">
-							{t('dashboard.stake.tabs.trading-rewards.fees-paid', { EpochPeriod: period })}
+						<div>
+							<div className="title">
+								{t('dashboard.stake.tabs.trading-rewards.trading-activity-reset')}
+							</div>
+							<div className="value">
+								{resetTime > new Date().getTime() / 1000
+									? formatTruncatedDuration(resetTime - new Date().getTime() / 1000)
+									: t('dashboard.stake.tabs.trading-rewards.pending-for-rewards')}
+							</div>
 						</div>
-						<div className="value">{formatDollars(totalFuturesFeePaid, { minDecimals: 2 })}</div>
-					</div>
-					{showEstimatedValue ? (
-						<>
+					</CardGrid>
+					<FlexDivRow>
+						<Button
+							fullWidth
+							variant="flat"
+							size="sm"
+							onClick={handleClaim}
+							disabled={claimDisabled}
+						>
+							{t('dashboard.stake.tabs.trading-rewards.claim')}
+						</Button>
+					</FlexDivRow>
+				</CardGridContainer>
+				<CardGridContainer>
+					<CardGrid>
+						<CustomStyledTooltip
+							preset="bottom"
+							width="260px"
+							height="auto"
+							content={t('dashboard.stake.tabs.trading-rewards.trading-rewards-tooltip')}
+						>
 							<div>
-								<div className="title">
-									{t('dashboard.stake.tabs.trading-rewards.estimated-rewards')}
-								</div>
-								<LogoText yellow>{truncateNumbers(wei(estimatedReward), 4)}</LogoText>
+								<WithCursor cursor="help">
+									<div className="title">
```

Comment:
Could we update to styled and maybe a base component instead of using className on these?

Reply:
I've updated this to a styled component. In the next phase, I'm going to add a color prop to the `Body` and `Heading` components. That will help in cases like this where we only need to set the color.